src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -70,9 +65,6 @@ public class NodeSettings
         /// <summary>The node's user agent.</summary>
         public string Agent { get; private set; }
```

Comment:
I think this moved to ConnectionManagerSettings

Reply:
See above.